_data/simple-icons.json

```diff
@@ -584,7 +584,7 @@
         {
             "title": "Apache Cassandra",
             "hex": "1287B1",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/5/5e/Cassandra_logo.svg"
+            "source": "https://commons.wikimedia.org/wiki/File:Cassandra_logo.svg"
```

Comment:
I'm wondering if we should use the official Apache source instead like the other Apache logos: https://apache.org/logos/#cassandra

Reply:
in the mean time, let's change the source to that one.

Reply:
changed the source as suggested